dom/src/main/scala-2/org/scalajs/dom/EncapsulationMode.scala

```diff
@@ -0,0 +1,11 @@
+package org.scalajs.dom
+
+import scala.scalajs.js
+
+@js.native
+sealed trait EncapsulationMode extends js.Any
```

Comment:
Sorry to nitpick these names, in most cases it makes sense to stay consistent with names used in the whatwg spec and Typescript definitions. Thanks!

```suggestion
sealed trait ShadowRootMode extends js.Any
```